app/data/AnalysisPlatformDS/instances/AnalysisPlatform.json

```
@@ -3,5 +3,24 @@
   "type": "/models/AnalysisPlatform",
   "label": "Analysis Platform App",
   "name": "analysis-platform",
-  "dataSources": ["AnalysisPlatformDS"]
+  "dataSources": [
+    "AnalysisPlatformDS"
+  ],
+  "assets": {
+    "type": "/models/AssetList",
+    "assets": [
+      {
+        "_id": "4483c9b0-d505-46c9-a157-94c79f4d7a6b",
+        "type": "/models/Asset",
+        "name": "exampleAsset"
+      }
+    ]
+  },
```

Looks like this is a bit wrong compared to the AnalysisPlatform blueprint?  I believe this is more correct (same structure as the analyses list below)

```suggestion
    "assets": [
      {
        "_id": "4483c9b0-d505-46c9-a157-94c79f4d7a6b",
        "type": "/models/Asset",
        "name": "exampleAsset"
      }
    ]
```

---

This is a workaround for having plugins that work on "list-of-things". Will discuss later today

---

decided in meeting that this is okay for now, but we might need to experiment to find a solution to make UI plugins handle lists